package.json

```diff
@@ -628,9 +637,14 @@
                     "group": "1@1"
                 },
                 {
-                    "command": "appService.RemoveCosmosDBConnection",
+                    "command": "appService.RevealConnectionInAppSettings",
                     "when": "view == azureAppService && viewItem == cosmosDBConnection",
                     "group": "2@1"
+                },
+                {
+                    "command": "appService.RemoveCosmosDBConnection",
```

Comment:
Adding 'remove' is not related to "Reveal in App Settings button". Please do a separate PR for that.

Reply:
It's already [in master](https://github.com/Microsoft/vscode-azureappservice/blob/3485c7b996f951969ccae5c3db5fac16dcffe83d/package.json#L630-L634). Github just shows changes not in the best way